src/page.ts

```diff
@@ -114,6 +124,39 @@ export class Page extends EventTarget {
       );
     });
 
+    if (options?.sandbox) {
+      const { sandbox, sandboxPrompt } = options;
+      this.#celestial.Fetch.enable({});
+      this.#celestial.addEventListener("Fetch.requestPaused", async (e) => {
+        const { request, requestId, resourceType } = e.detail;
+        if (
+          (sandbox === true) ||
+          (Array.isArray(sandbox) &&
+            sandbox.includes(resourceType))
+        ) {
+          const url = new URL(request.url);
```

Comment:
Are these always absolute URLs? If not, we should set the base.

Reply:
I need to check as the [`Network.Request`](https://chromedevtools.github.io/devtools-protocol/tot/Network/#type-Request) doc doesn't precise, but think they are

Reply:
After playing a bit with this, seems it's always absolute (like `fetch("/test")` will be resolved related to the current domain page)